docker/dockerfile-generator.sh

```diff
@@ -324,7 +324,7 @@ printUserCreate(){
   echo "
 ARG HostUID
 ENV HostUID=\$HostUID
-RUN useradd -u \$HostUID -ms /bin/bash build
+RUN useradd -o -u \$HostUID -ms /bin/bash build
```

Comment:
I don't think it's a good idea to allow for non-unique IDs here. In particular, allowing UID=0 to be non-unique. The fix is to not build with as root.